src/test/java/ui/UITests.java

```diff
@@ -0,0 +1,111 @@
+package ui;
+
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+import ui.helpers.ActionsHelper;
+import ui.pages.DemoApiTestsOneLaunchPage;
+import ui.pages.DemoDashboardPage;
+import ui.pages.LaunchPage;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static ui.logger.UILogger.LOGGER;
+
+
+public class UITests extends BaseUITest {
+    private LaunchPage launchPage = new LaunchPage();
+    private DemoApiTestsOneLaunchPage demoApiTestsOneLaunchPage = new DemoApiTestsOneLaunchPage();
```

Comment:
Used in only one method, can be converted to local variable.

Reply:
"+"